There seem to be issues inside the type library that make it not compile.

When trying to use the types without setting `skipLibCheck` to `true`, I get the following errors inside the type library itself:

```
/home/johannesloher/Development/foundry-vtt/systems/ds4/node_modules/foundry-vtt-types/types/augments/PIXI.d.ts(1,25): error TS2792: Cannot find module 'pixi.js'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?
/home/johannesloher/Development/foundry-vtt/systems/ds4/node_modules/foundry-vtt-types/types/augments/handlebars.d.ts(2,1): error TS2303: Circular definition of import alias 'module'.
/home/johannesloher/Development/foundry-vtt/systems/ds4/node_modules/foundry-vtt-types/types/augments/howler.d.ts(5,22): error TS2694: Namespace '"/home/johannesloher/Development/foundry-vtt/systems/ds4/node_modules/foundry-vtt-types/types/augments/howler"' has no exported member 'Howl'.
/home/johannesloher/Development/foundry-vtt/systems/ds4/node_modules/foundry-vtt-types/types/augments/utility-types.d.ts(5,74): error TS2694: Namespace '"/home/johannesloher/Development/foundry-vtt/systems/ds4/node_modules/foundry-vtt-types/types/augments/utility-types"' has no exported member 'Optional'.
/home/johannesloher/Development/foundry-vtt/systems/ds4/node_modules/foundry-vtt-types/types/framework/entities/item.d.ts(81,7): error TS2502: 'actor' is referenced directly or indirectly in its own type annotation.
```
I am using Typescript 4.1.3. Here is my `tsconfig.json`:

```json
{
    "compilerOptions": {
        "target": "ES2017",
        "lib": ["DOM", "ES6", "ES2017"],
        "types": ["foundry-vtt-types"],
        "esModuleInterop": true,
    }
}
```

Am I doing anything wrong? Should the types "just work"?

using `moduleResolution: 'node'` get's rid of most of the problems. There is only one that stays:
```
/home/johannesloher/Development/foundry-vtt/systems/ds4/node_modules/foundry-vtt-types/types/framework/entities/item.d.ts(81,7): error TS2502: 'actor' is referenced directly or indirectly in its own type annotation.
```

--- a/types/applications/forms/item.d.ts
+++ b/types/applications/forms/item.d.ts
@@ -35,7 +35,7 @@ declare class ItemSheet<T = object, O extends Item = Item, F = object> extends B
    * The Actor instance which owns this item. This may be null if the item is
    * unowned.
    */
-  get actor(): Actor;
+  get actor(): Actor<O, Actor.Data<any, Actor.OwnedItem<O>>> | null;
 
   /**
    * Activate listeners which provide interactivity for item sheet events

--- a/types/framework/entities/item.d.ts
+++ b/types/framework/entities/item.d.ts
@@ -84,7 +84,7 @@ declare class Item<D extends Item.Data = Item.Data<any>> extends Entity<D> {
   /**
    * A convenience reference to the Actor entity which owns this item, if any
    */
-  get actor(): Actor | null;
+  get actor(): Actor<this, Actor.Data<any, Actor.OwnedItem<this>>> | null;
 
   /**
    * A convenience reference to the image path (data.img) used to represent this Item

